display_list/display_list_unittests.cc

```diff
@@ -387,7 +386,6 @@ TEST_F(DisplayListTest, BuildRestoresAttributes) {
   builder.Build();
   check_defaults(builder, cull_rect);
 
-  receiver.setDither(true);
```

Comment:
The two lines following that should also be removed. They have nothing to check if the receiver wasn't modified.